### PR TITLE
i18n(es): update `site-search`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes # N/A
- What does this PR change? Give us a brief description.
Updates `site-search

### CODE REVIEW

File: docs/src/content/docs/es/guides/site-search.mdx [hunk 1]
```diff
@@ -162,3 +167,4 @@ Para agregar traducciones de la UI al modal en otros idiomas, usa el [sistema de
    	"docsearch.noResultsScreen.reportMissingResultsLinkText": "Let us know."
    }
    ```
+</Steps>
```
Comment (liruifengv) [inline docs/src/content/docs/es/guides/site-search.mdx:170]:
```suggestion

</Steps>
```
Hi @dreyfus92 There need a line break. see: https://github.com/withastro/starlight/pull/1647

File: docs/src/content/docs/es/guides/site-search.mdx [hunk 2]
```diff
@@ -102,13 +104,16 @@ Si tienes acceso al [programa DocSearch de Algolia](https://docsearch.algolia.co
...
+
 Con esta configuración actualizada, la barra de búsqueda de tu sitio ahora abrirá un modal de Algolia en lugar del modal de búsqueda predeterminado.
 
 #### Traduciendo la UI de DocSearch
 
 DocSearch solo proporciona cadenas de UI en inglés de forma predeterminada.
 Para agregar traducciones de la UI al modal en otros idiomas, usa el [sistema de internacionalización](/es/guides/i18n/#traduce-la-ui-de-starlight) incorporado de Starlight.
 
+<Steps>
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/site-search.mdx:116]:
```suggestion
<Steps>

```

This one will actually also need an empty line to not break when `pnpm format` runs.

**Review by HiDeoo — CHANGES_REQUESTED** [review]

**Comment by dreyfus92** [comment]
Thank you both for your suggestions.

**Review by HiDeoo — APPROVED** [review]
Thanks for the update 🙌

**Pull request merged**